src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
```

Comment:
Good job importing only the classes that you need as it makes the class easier to comprehend and maintain.

Reply:
thanks!